Services/CO.CDP.OrganisationInformation.Persistence/IOrganisationRepository.cs

```diff
@@ -15,6 +15,8 @@ public interface IOrganisationRepository : IDisposable
 
     public Task<OrganisationPerson?> FindOrganisationPerson(Guid organisationId, string userUrn);
 
+    public Task<Person?> FindPersonWithTenant(Guid personId);
```

Comment:
OrganisationRepository doesn't seem like the right place to look for persons.

Reply:
I thought the same yesterday evening. Moved to IPersonRepository.